onnxruntime/core/graph/model.h

```diff
@@ -244,8 +244,8 @@ class Model {
   // properties that would normally come from ModelProto
   std::string producer_version_;
   std::string producer_name_;
-  int64_t model_version_ = 0;
-  int64_t ir_version_ = kNoVersion;
+  int64_t model_version_ = kNoVersion;
+  int64_t ir_version_ = 0;
   std::string domain_;
```

Comment:
Both could use kNoVersion as that's what Model::ModelVersion and Model::IrVersion potentially return